### PR TITLE
refactoring

### PR DESCRIPTION
**Pull request by fumiya-kume**

### CODE REVIEW

**Comment by Hunachi** [comment]
LGTM（※細かいコードの変更については確認してません）

File: android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/wake/autowake/AutoWakeService.kt [hunk 1]
```diff
@@ -1,10 +1,15 @@
 package jp.hacks.smartbread.ui.main.wake.autowake
 
 import android.content.Context
```
Comment (fumiya-kume) [inline android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/wake/autowake/AutoWakeService.kt:3]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jphacks/FK_1908/blob/90f178290d8f851b5ea294f652a224385d2a1947/android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/wake/autowake/AutoWakeService.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>

File: android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/wake/autowake/AutoWakeService.kt [hunk 2]
```diff
@@ -23,9 +29,15 @@ internal class AutoWakeService(
             ticker.consumeEach {
                 val targetDate = wakeDateReadonlyRepository.loadWakeDate()
                 val now = Date()
+                // ここを専用タイマークラスに変更する
                 if (now.hours == targetDate.hours && now.minutes == targetDate.minutes) {
                     burnBreadUsecase.startBurn()
                 }
+                if(now.hours == 22 && now.minutes == 0){
```
Comment (fumiya-kume) [inline android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/wake/autowake/AutoWakeService.kt:36]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jphacks/FK_1908/blob/90f178290d8f851b5ea294f652a224385d2a1947/android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/wake/autowake/AutoWakeService.kt#L36"></span>Missing spacing after “if”</td>
    </tr>
  </tbody>
</table>

Comment (fumiya-kume) [inline android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/wake/autowake/AutoWakeService.kt:36]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jphacks/FK_1908/blob/90f178290d8f851b5ea294f652a224385d2a1947/android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/wake/autowake/AutoWakeService.kt#L36"></span>Missing spacing before “{“</td>
    </tr>
  </tbody>
</table>

File: android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/wake/autowake/AutoWakeService.kt [hunk 3]
```diff
@@ -23,9 +29,15 @@ internal class AutoWakeService(
...
+                    GlobalScope.launch(Dispatchers.IO) {
+
```
Comment (fumiya-kume) [inline android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/wake/autowake/AutoWakeService.kt:38]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jphacks/FK_1908/blob/90f178290d8f851b5ea294f652a224385d2a1947/android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/wake/autowake/AutoWakeService.kt#L38"></span>Unexpected blank line(s) before “}”</td>
    </tr>
  </tbody>
</table>

**Pull request merged**